### PR TITLE
22289-Update Text in French

### PR DESCRIPTION
**Pull request by hanlunBCRegistries**
Issue: #248 
Issue: /bcgov/entity#22289

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-ar-dev--pr-280-81xn4yr4.web.app

File: web/site/content/fr-CA/help.md [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
+- Coordonnées : fournissez un numéro de téléphone et une adresse électronique à utiliser pour le nouveau compte.
+
+## Rapport annuel
+Vous êtes maintenant prêt à déposer votre rapport annuel. Vous arriverez sur une page pré-remplie avec les détails de votre entreprise. Vous serez invité à :
+- Fournir le statut de votre assemblée générale annuelle (AGA). Les options suivantes sont proposées :
+  - Notre AGA a eu lieu - si sélectionné, vous serez invité à fournir la date à laquelle l'AGA a eu lieu.
+  - Notre AGA doit se tenir
+  - Le conseil a voté pour ne pas tenir d'AGA
+- Pour plus d'informations sur les assemblées générales annuelles, veuillez consulter la section ci-dessous.
+- Examiner et confirmer les adresses et les administrateurs avant de certifier et de déposer votre rapport annuel. Si vous devez mettre à jour l'adresse de votre entreprise et/ou les administrateurs, vous devrez vous rendre sur [Corporate Online](https://www.corporateonline.gov.bc.ca) pour effectuer ces modifications d'abord.
```
Comment (deetz99) [inline web/site/content/fr-CA/help.md:31]: Should this open in a new tab?

File: web/site/content/fr-CA/help.md [hunk 2]
```diff
@@ -1,48 +1,48 @@
...
+- Nominations de l'auditeur (sauf renonciation par résolution spéciale)
+- Élection des administrateurs
+- Autres questions comme indiqué dans l'avis de l'AGA
+
+## Obligation de tenir une AGA
+Les entreprises enregistrées en Colombie-Britannique sont tenues de tenir une AGA au moins une fois par an et pas plus de 15 mois après la date anniversaire de l'enregistrement de votre entreprise en Colombie-Britannique.
+
+Si votre entreprise est nouvellement enregistrée en Colombie-Britannique, votre première AGA doit avoir lieu au plus tard 18 mois après la date de son enregistrement en Colombie-Britannique.
+
+Le rapport annuel demande le statut de votre AGA pour aider votre entreprise à rester conforme aux exigences de la [Loi sur les sociétés de la Colombie-Britannique](https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/02057_00_multi).
```
Comment (deetz99) [inline web/site/content/fr-CA/help.md:45]: Should this open in a new tab?

Comment (hanlunBCRegistries) [reply web/site/content/fr-CA/help.md]: That is correct, just updated it

**Review by thorwolpert — COMMENTED** [review]
Couple of things
We try to use https://www.conventionalcommits.org/en/v1.0.0/ for commit messages.
PR should reference the issue it is addressing
We generally don't include changes that are just formatting (eg. pnpm-lock.yml) where nothing has changed

**Comment by Maribeth-Wilson** [comment]
reached to Masha Kaur of GDX. Will post response here when available.

**Comment by ozamani9gh** [comment]
the work was done on 22289

**Review by EPortman — APPROVED** [review]

**Pull request merged**